roxctl/central/generate/volumes.go

```diff
@@ -58,9 +53,8 @@ func noVolume(cliEnvironment environment.Environment) *cobra.Command {
 
 func hostPathVolume(cliEnvironment environment.Environment) *cobra.Command {
```

Comment:
Do we still support `hostPathVolume`?

Reply:
Yes, it still handles the db hostpath. We are going to deprecate it eventually but not with this PR.